godot-core/src/builtin/string/gstring.rs

```diff
@@ -123,6 +131,27 @@ impl GString {
         std::slice::from_raw_parts(ptr as *const char, len as usize)
     }
 
+    /// Gets the internal chars slice from a [`GString`].
+    ///
+    /// # Safety
+    ///
+    /// - Godot ensures that the string is a valid UTF-32.
+    #[cfg(since_api = "4.1")]
+    pub fn chars(&self) -> &[char] {
```

Comment:
why are we reimplementing `chars_unchecked` here? Couldn't we instead just call `chars_unchecked()` in this method?